src/cloud/components/RateLimitAlert.tsx

```diff
@@ -89,11 +90,16 @@ const RateLimitAlert: FC<Props> = ({alertOnly, className, location}) => {
         )
       }
     }
-  }, [showUpgrade, status])
+  }, [showUpgrade, status]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
Why is this lint override needed? It doesn't look like any of the related code changed, so I'm curious why the linter would start complaining suddenly.

Reply:
It has always complained. No one else minded. But I mind. 😄